core/src/tx.rs

```diff
@@ -229,7 +229,7 @@ impl TransactionExecutor {
             ));
         }
 
-        debug!("Validating transaction: {:?}", tx);
+        debug!(tx=%tx.as_ref().hash(), "Validating transaction");
```

Comment:
Does `rustfmt` complain about whitespace in these macros?

Reply:
Not really, it's ok with both options

Reply:
It's custom syntax, so it doesn't touch it